.github/test-configuration.json

```diff
@@ -15,7 +15,7 @@
   },
   "ndc-postgres-12": {
     "feature": "postgres",
-    "filter": "not test(configure_initial_configuration_is_unchanged) and not test(configure_is_idempotent)",
+    "filter": "not test(postgres_current_only)",
```

Comment:
@danieljharvey : How does this look to you?

Reply:
Looks great - naming it like this explains why we're skipping too which is nice.